src/libraries/System.Net.Http/src/System/Net/Http/HttpRequestException.cs

```diff
@@ -28,6 +28,23 @@ public HttpRequestException(string message, Exception inner)
             }
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="HttpRequestException" /> class with a specific message that describes the current exception, an inner exception, and an HTTP status code.
+        /// </summary>
+        /// <param name="message">A message that describes the current exception.</param>
+        /// <param name="inner">The inner exception.</param>
+        /// <param name="statusCode">The HTTP status code.</param>
```

Comment:
```suggestion
        /// <param name="statusCode">If this exception was sourced from a HTTP status code, the status code. Otherwise, null.</param>
```

Would be good to indicate when supplying a status code is appropriate.

Reply:
"sourced from" seems weird to me.

Reply:
>"sourced from" seems weird to me.

I'm not a fan of that phrase either.

However, the final edits to the documentation will happen after this XML gets ported into the docs.microsoft.com tool for the API documentation. A PR will get created in the [dotnet-api-docs](https://github.com/dotnet/dotnet-api-docs)  repo and our documentation content editors will check the final phrasing to make sure it confirms to existing technical documentation style we have.

Reply:
I've tweaked this slightly differently by mentioning the nullability in the docs for the property's value. I don't feel like the constructor param is quite the best place for this, particularly since we don't discuss the nullability of any other params (e.g. the inner exception), or in other similar classes that I could find.